app/views/foods/index.html.erb

```diff
@@ -0,0 +1,28 @@
+  <h1>Search results</h1>
+
+  <table class="table table-hover">
```

Comment:
I appreciate the formatting done here to present the data in a more readable way!

Reply:
It is just missing the "total foods" part of the user story